api/docs/release.dox

```diff
@@ -141,6 +141,12 @@ Further non-compatibility-affecting changes include:
    instructions in the trace. This works for traces that have embedded instruction
    encodings in them, and also for legacy traces without embedded encodings where the
    encodings are obtained from the application binaries instead.
+ - Added noise_generator_t scaffolding as a reader_t to produce synthetic trace records
```

Comment:
As in all doxygen content, make any type a link with # (may have to fully qualify as well).  Two types here.

Reply:
Done.